marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/classifier/TextClassifier.java

```diff
@@ -51,17 +63,23 @@ public TextClassifier(String host, String https, String port, String endpoint, S
         classificationClient.setClassificationConfiguration(classificationConfiguration);
     }
 
-    public Map<String, Collection<ClassificationScore>> classifyText(String sourceUri, String text) {
+    public Document classifyTextToXml(String sourceUri, String text) {
         try {
-            long start = System.currentTimeMillis();
-            Map<String, Collection<ClassificationScore>> results = classificationClient.getClassifiedDocument(new Body(text), new Title("")).getAllClassifications();
-            if (Util.CLASSIFIER_LOGGER.isDebugEnabled()) {
-                Util.CLASSIFIER_LOGGER.debug("Source URI: {}; count of results: {}; time: {}", sourceUri, results.size(),
-                    (System.currentTimeMillis() - start));
-            }
-            return results;
+            byte[] rawResponse = classificationClient.getClassificationServerResponse(new Body(text), new Title(sourceUri));
+            InputSource response = new InputSource(new StringReader(new String(rawResponse, StandardCharsets.UTF_8)));
+            return builder.parse(response);
         } catch (ClassificationException e) {
-            throw new MarkLogicLangchainException(String.format("Unable to generate concepts for: %s; cause: %s", text, e.getMessage()), e);
+            throw new MarkLogicLangchainException(String.format("Unable to generate classification for: %s; cause: %s", text, e.getMessage()), e);
```

Comment:
I don't think we want to include the text in the error; the text could be of any length. Including the sourceUri should be helpful enough. 

And I don't think two catch blocks are needed. We don't want to tell a connector or Flux user that we couldn't parse the classification response as XML; there's nothing they can do about that. Just abstract it as "Unable to classify data from document with URI: sourceUri; cause: the error message". A Flux user can use `--stacktrace` to see the complete error.